{
  willThrow(SomethingWasWrong.class,AnotherThingWasWrong.class).given(mock).voidMethod();
  try {
    mock.voidMethod();
    fail();
  }
 catch (  SomethingWasWrong expected) {
  }
}
